doc/source/whats_new.rst

```diff
@@ -10,6 +10,8 @@ Current
 
 Changelog
 ~~~~~~~~~
+    - Add Xdawn algorithm for denoising and decoding of ERP/ERF by `Alexandre
+    Barachant`_
```

Comment:
Actually, you haven't provided your homepage information. So, the link won't render correctly